src/ServiceControl.Transports.ASB/packages.config

```diff
@@ -1,10 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Newtonsoft.Json" version="11.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="7.0.1" targetFramework="net452" />
-  <package id="NServiceBus.Azure.Transports.WindowsAzureServiceBus" version="8.0.0" targetFramework="net452" />
-  <package id="NServiceBus.CustomChecks" version="3.0.0" targetFramework="net452" />
-  <package id="NServiceBus.Newtonsoft.Json" version="2.1.0" targetFramework="net452" />
-  <package id="NServiceBus.Raw" version="3.0.0" targetFramework="net452" />
```

Comment:
Do the transitive dependencies automatically take NSB.Raw into account? I'm asking because the transport customization needs it

Reply:
Correct, every package reference in the main project that isn't marked with `PrivateAssets="All"` will be transitively referenced through the project reference, so there's no need to reference it again.

Reply:
If the customization needs it, isn't it better to have an explicit reference to Raw?

Reply:
No, I don't think so. It's much better to have a single reference, which can then be updated in a single place, ensuring version unification.